src/generic_factory.h

```diff
@@ -985,4 +986,65 @@ typename std::enable_if<std::is_constructible<T, std::string>::value, bool>::typ
     return res;
 }
 
+inline bool assign( JsonObject &jo, const std::string &name, units::volume &val,
+                    bool strict = false,
+                    const units::volume lo = units::volume( std::numeric_limits<units::volume::value_type>::min(),
+                            units::volume::unit_type{} ),
+                    const units::volume hi = units::volume( std::numeric_limits<units::volume::value_type>::max(),
+                            units::volume::unit_type{} ) )
+{
+    // Currently JSON data contains volume in 250ml units.
+    // TODO: change JSON to contain milliliter values.
```

Comment:
Do you mean to use `///` instead of plain `//` so Doxygen extracts this?

> once we have 100% JSON linter coverage that transition could be easily performed via  jq

Yes, but external mods will *silently* break (items will have a wrong volume). I'd prefer to have a new member and falling back to the current system if the new member is not present. Anyway, it's not part of this PR.